pkg/testutil/test/utilities.go

```diff
@@ -0,0 +1,42 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package test
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"os"
+)
+
+// IsRoot returns true if we are root... simple
+func IsRoot() bool {
```

Comment:
Why can’t we use pkg/rootlessutil.IsRootless

Reply:
We could.

This is currently used solely to check for a corner case of Docker.

https://github.com/containerd/nerdctl/blob/192d1cb0d5caab1794b3f0608a0660d64e4fc31e/cmd/nerdctl/volume/volume_inspect_test.go#L54

In that context, rootlessutil.IsRootless seems a bit misleading.

Note that this `IsRoot` needs to go eventually - it is not even proper to test the case in question with Docker - what we really want to test is if Docker Daemon is running root, while the cli is not.

Reply:
“IsRoot” sounds misleading too. I’d rather suggest just calling os.Geteuid directly

Reply:
Will amend.

Reply:
Done.